README.md

```diff
@@ -7,9 +7,9 @@ Instructions on how to do this would get overly complex, which is why we opted t
 
```

Comment:
for the sake of accuracy you should change "... with Sops Go plugin from https://github.com/Agilicus/kustomize-sops" by "... with Sops Exec plugin from https://github.com/goabout/kustomize-sopssecretgenerator"

Reply:
OK.